interview-questions.md

```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Interview questions are an opportunity for you to practice having a conversation with an interviewer. Try to share as much content as reasonably possible. Extra content can be your experience with the tool or giving an example of the tool. You can also discuss some new information you discovered from your research.